ompi/mpi/fortran/use-mpi-f08/attr-fn-f08-callback-interfaces.h

```diff
@@ -43,7 +45,7 @@ interface
 
   subroutine MPI_COMM_NULL_COPY_FN( comm, comm_keyval, extra_state, &
                                     attribute_val_in, attribute_val_out, &
-                                    flag, ierr )
+                                    flag, ierr ) BIND(C,name="ompi_comm_null_copy_fn_f")
```

Comment:
Can you explain why these subroutines need to be BIND(C) to a specific C function name?